src/NUnitFramework/framework/Attributes/TypeGetMemberExtension.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace NUnit.Framework
+{
+    internal static class TypeGetMemberExtension
```

Comment:
Thanks for making these extensions internal visibility by default.

Suggestion: I think there may be an existing + recently created [`TypeExtensions`](https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Internal/Extensions/TypeExtensions.cs) class which lives in `Internal/Extensions`. Would it make sense for this new method to live there too?

Reply:
I had looked for extension classes, but missed this one. Method moved and Tests added.